openid-sharedsignals-framework-1_0.md

```diff
@@ -762,6 +764,14 @@ which can be used by Event Receivers to create and delete one or more Event Stre
 The API can also be used to query and update the Event Stream's configuration and status,
 add and remove Subjects, and trigger verification for those streams.
 
+Unless there exists some other method of establishing trust between a Transmitter and
```

Comment:
https://github.com/openid/sharedsignals/pull/134/files adds TLS language to interop spec, but 👍 to bring it here.

Though, I feel the last sensence around `authorization schemes` is either duplicate of what is already referenced above or misplaced.